Source/WebCore/Modules/webcodecs/WebCodecsAudioEncoder.cpp

```diff
@@ -137,7 +137,7 @@ ExceptionOr<void> WebCodecsAudioEncoder::configure(ScriptExecutionContext&, WebC
     if (m_internalEncoder) {
         queueControlMessageAndProcess({ *this, [this, config]() mutable {
             m_isMessageQueueBlocked = true;
-            m_internalEncoder->flush([weakThis = ThreadSafeWeakPtr { *this }, config = WTFMove(config)]() mutable {
+            m_internalEncoder->flush([weakThis = ThreadSafeWeakPtr { *this }, config = WTFMove(config).isolatedCopy()]() mutable {
```

Comment:
why WTFMove?

Reply:
Ditto, It allows to use isolatedCopy() && which might save us a string copy if the string ref count is 1.